src/Miningcore/Blockchain/Ethereum/EthereumPayoutHandler.cs

```diff
@@ -345,6 +347,12 @@ public async Task PayoutAsync(Balance[] balances, CancellationToken ct)
                     break;
                 }
 
+                if(balance.Amount.CompareTo(maxPayout) > 0)
```

Comment:
balance.Amount > maxPayout is enough as well. Isn't we should compare against min threshold?

Reply:
the min threshold is handled by the DB query for automatic payments.
for force payments it's done in the function, since that's where we filter amounts